pkg/components/minio/minio.go

```diff
@@ -9,28 +9,33 @@ import (
 	"github.com/mattermost/mattermost-operator/pkg/components/utils"
 	mattermostApp "github.com/mattermost/mattermost-operator/pkg/mattermost"
 
-	minioOperator "github.com/minio/minio-operator/pkg/apis/miniocontroller/v1beta1"
-
+	ptrUtil "github.com/mattermost/mattermost-operator/pkg/utils"
+	minioOperator "github.com/minio/operator/pkg/apis/minio.min.io/v2"
 	corev1 "k8s.io/api/core/v1"
 	resource "k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	MinioNameAffix = "-minio-v4"
```

Comment:
Can it be called suffix? 0/5

Reply:
Honestly I don't know why I used affix here. 😅